docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -89,44 +83,58 @@ Next, you need to remove the empty users from your MySQL database:
     mysql> delete from mysql.user where user = '' ;
     mysql> flush privileges;
 
-Step 2 : Server configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The others Servers configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-First, you need to make sure the interfaces name will be the same on all servers. See 'Setting the interfaces name on CentOS 7' in the Appendix section of this document if all your servers don't already have the same interface names.
+First, you need to make sure the interfaces name will be the same on *all servers*. See 'Setting the interfaces name on CentOS 7' in the Appendix section of this document if all your servers don't already have the same interface names.
 
 Next, you will need to configure each server so the services can bind on IP addresses they don't currently have configured. This allows faster failover of the services.
 
-On all your servers, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
+On *all your servers*, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
 
   net.ipv4.ip_nonlocal_bind = 1
 
 Then, disable the default MariaDB systemd service unit as PacketFence already provides one (packetfence-mariadb)
 
   # systemctl disable mariadb
 
-Now, on the first server, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. What matters is that the path to the PEM file should be /usr/local/pf/conf/ssl/server.pem since HAProxy will expect it to be there.
+Now, on *the first server*, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. What matters is that the path to the PEM file should be /usr/local/pf/conf/ssl/server.pem since HAProxy will expect it to be there.
 
   # cd /usr/local/pf/conf/ssl
   # cat server.key server.crt > server.pem
 
-Going through the configurator
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Basic PacketFence configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Now, on the first server of your cluster, you should go through the configurator, configuring the server as if it was in standalone (no HA interfaces). You should leave the services stopped at the end of the configurator.
+Now, on *the first server* of your cluster, you should go through the configurator, until *step 6* (Fingerbank configuration). You should leave the services stopped at the end of the configurator.
```

Comment:
Your removed the step numbers from the headers, so step 6 doesn't make sense anymore

Reply:
This is the Step 6 of the PF Configurator (FingerBank config), not the ex-Step6.